### ISSUE TITLE
Other Work this Sprint (Sprint #13)

### ISSUE THREAD

**Issue by schancase** [cds-snc/notification-planning#2701]
This is our "Other Work Doing or Done" Bucket will be created on a per-Sprint or multi-sprint Basis in order to:

* Bring visibility at the feature level to other work we are doing;
* Prompt us to examine the list of child items (i.e. Tasks) to see if in fact they should be parented elsewhere OR whether a bunch of them on their own merit their own feature; and
* Improve workflow metrics for showing work the team got done each sprint at the feature level

**Comment by schancase** [comment]
closing as done for this sprint. All undone tasks were parented to #2814
